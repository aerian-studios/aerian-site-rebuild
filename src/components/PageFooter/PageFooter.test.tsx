/// <reference types="@types/jest" />
import * as React from "react";

import * as renderer from "react-test-renderer";

import { PageFooter } from "./index";

describe("PageFooter", () =>
    it("renders correctly", () => {
        const tree = renderer
            .create(<PageFooter className="myClass" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
