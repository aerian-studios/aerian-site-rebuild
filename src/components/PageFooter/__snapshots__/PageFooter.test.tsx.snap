// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter renders correctly 1`] = `
<footer
  className=" myClass block--full"
>
  <div />
</footer>
`;
